src/main/java/com/malt/aster/activities/uno/UnoMainGame.java

```diff
@@ -153,13 +155,26 @@ public void handlePrivateMessage(PrivateMessageReceivedEvent evt) {
                 int cardIndex = Integer.parseInt(evt.getMessage().getContentRaw()) - 1;
 
                 if (validChoice(sender, cardIndex)) {
```

Comment:
Plenty of if/else statements follow from around here - is this intended behaviour?